src/popup/router/components/AccountSwitcher.vue

```diff
@@ -147,17 +147,19 @@ export default {
   .swiper-button {
     position: absolute;
     opacity: 0.5;
-    top: 81px;
+    top: 61px;
```

Comment:
I think placing thing in the middle is better to accomplish with `top: 50%` and then `transform: translateY(-50%)`.

Reply:
we can't use `top: 50%` here, because the swiper button has a specific height.

Reply:
I'll try to do it on my side.